core/controllers/editor.py

```diff
@@ -780,6 +780,7 @@ class LearnerAnswerInfoHandler(EditorHandler):
     """Handles the learner answer info for an exploration state."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    IS_LOGGED_IN_CHECK_REQUIRED = False
 
     @acl_decorators.can_edit_entity
```

Comment:
I think wer had planned to change the decorator completely, right?

OR change the frontend such that it doesn't make a call to this handler if not logged in.

Reply:
Ignore this comment as I read @anubhavsinha98's comment on the issue thread after this review!